jobserver/templates/login.html

```diff
@@ -4,7 +4,7 @@
 <div class="container">
   <div class="row">
     <div class="col-sm-12">
-      <form method="POST" action="{% url "auth-login" "github" %}?next={{ request.path }}">
+      <form method="POST" action="{% url "auth-login" "github" %}?next={{ request.GET.next|default:"/" }}">
```

Comment:
This is a bit of a nitpick, but I'd be inclined to use a context variable here (`next_url` or something) which gets set in the view code rather than reference `request.GET.next` directly in the template. As it happens, the value of `GET.next` has already been validated, but it's not immediately obvious from looking at the template that that's the case; and in general referencing values directly from the request like this is a bit of a smell. It would also means you could do the default handling in the view as well, which would be a bit cleaner.

It should just be a case of doing something like this in the view:
```py
    def get(self, request, *args, **kwargs):
        next_url = request.GET.get("next", "/")
        if is_safe_path(next_url):
            return render(request, self.template_name, {"next_url": next_url})
        else:
            return bad_request(request, SuspiciousOperation)
```
```

Reply:
Ah! Great, thank you. I was going to ask about how to handle the default, but forgot. I've modified `Login.get` and *jobserver/templates/login.html*.